Gordon360/Models/CCT/Housing/Preference.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("ApplicationID", "Preference1")]
+[Table("Preference", Schema = "Housing")]
+public partial class Preference
+{
+    [Key]
+    [StringLength(255)]
+    [Unicode(false)]
+    public string ApplicationID { get; set; }
+
+    [Key]
+    [Column("Preference")]
+    [StringLength(255)]
+    [Unicode(false)]
+    public string Preference1 { get; set; }
```

Comment:
What is preference here? ID, text, etc? It could be a bit more descriptive of what data it is supposed to hold.

Also, why is there a 1 at the end?

Reply:
Yes, it stores the applicant's preferences. I will add some comments for these files!